cmd/operator-controller/main.go

```diff
@@ -315,13 +315,14 @@ func main() {
 		os.Exit(1)
 	}
 
-	unpacker := &source.ContainersImageRegistry{
-		BaseCachePath: filepath.Join(cachePath, "unpack"),
-		SourceContextFunc: func(logger logr.Logger) (*types.SystemContext, error) {
+	imageCache := imageutil.BundleCache(filepath.Join(cachePath, "unpack"))
```

Comment:
We are using `unpackCacheBasePath := filepath.Join(cacheDir, "unpack")` also in catalogd main.go . we should use a constant or variable for `unpack`.

Reply:
These unpack paths are decided/defined/used purely in the context of each `main.go`. It's just a coincidence/convention that they are the same. But they don't have to be.